src/tls.rs

```diff
@@ -1,6 +1,55 @@
 use core::ffi::{c_char, CStr};
 use core::fmt::Debug;
 
+use esp_idf_sys::psk_hint_key_t;
+
+use crate::private::cstr::RawCstrs;
+
+#[derive(Copy, Clone, Eq, PartialEq)]
+pub struct Psk<'a> {
+    pub key: &'a [u8],
+    pub hint: &'a str,
+}
+
+impl<'a> Debug for Psk<'a> {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> Result<(), core::fmt::Error> {
+        f.debug_struct("Psk")
+            .field("hint", &self.hint)
```

Comment:
You sure that's OK?

Reply:
It should and apparently does according to my test with logging a `Psk` struct. This support is available in _core_ and gives a debug output like `Psk { hint: "my-hint-is-longer-than-yours", .. }`.